src/azure-cli/azure/cli/command_modules/dla/_client_factory.py

```diff
@@ -4,39 +4,40 @@
 # --------------------------------------------------------------------------------------------
 
 
+# pylint: disable=line-too-long,
 def cf_dla_account(cli_ctx, _):
     from azure.cli.core.commands.client_factory import get_mgmt_service_client
-    from azure.mgmt.datalake.analytics.account import DataLakeAnalyticsAccountManagementClient
+    from azure.cli.command_modules.dla.vendored_sdks.azure_mgmt_datalake_analytics.account import DataLakeAnalyticsAccountManagementClient
```

Comment:
It may be better to use relative import since `azure.cli.command_modules.dla.vendored_sdks` is in the same package `azure.cli.command_modules.dla` as `azure.cli.command_modules.dla._client_factory`.

This may also save us from adding `# pylint: disable=line-too-long`.

```suggestion
    from .vendored_sdks.azure_mgmt_datalake_analytics.account import DataLakeAnalyticsAccountManagementClient
```

Reply:
I've changed to relative import.
Weird, we hardly use relative import in azure-cli....